00-Introducing-HOOMD-blue/03-Initializing-the-System-State.ipynb

```diff
@@ -35,6 +35,17 @@
     "import numpy"
```

Comment:
<p>The rendered docs are included in the sphinx documentation. So we need the notebook run to succeed without errors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>